### PR TITLE
Use https for caniuse.com links

### PR DESCRIPTION
**Pull request by RonaldDijkstra**

### CODE REVIEW

File: source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown [hunk 1]
```diff
@@ -73,7 +73,7 @@ Now we add the styling. Give the faces a background color and make sure they are
 }
 ~~~
 
-__Note:__ Don't forget to add vendor prefixes for the `transform*` properties if you want cross-browser [support](http://caniuse.com/#feat=transforms2d){:target="_blank"}{:rel="noopener"}. In the demos I use [SCSS](http://sass-lang.com) and the [Bourbon](https://www.bourbon.io/){:target="_blank"}{:rel="noopener"} mixin library so I don't have to worry about these prefixes.
+__Note:__ Don't forget to add vendor prefixes for the `transform*` properties if you want cross-browser [support](https://caniuse.com/#feat=css-animation){:target="_blank"}{:rel="noopener"}. In the demos I use [SCSS](http://sass-lang.com) and the [Bourbon](https://www.bourbon.io/){:target="_blank"}{:rel="noopener"} mixin library so I don't have to worry about these prefixes.
```
Comment (sn3p) [inline source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown]: `css-animation` ? 🤔

File: source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown [hunk 2]
```diff
@@ -145,7 +145,7 @@ Now we can apply the animation to the cubes' faces. We configure the animation b
 }
 ~~~
 
-__Note:__ Don't forget to add vendor prefixes for `@keyframes` and `animation*` if you want cross-browser [support](http://caniuse.com/#feat=css-animation){:target="_blank"}{:rel="noopener"}.
+__Note:__ Don't forget to add vendor prefixes for `@keyframes` and `animation*` if you want cross-browser [support](https://caniuse.com/#search=css-animation){:target="_blank"}{:rel="noopener"}.
```
Comment (sn3p) [inline source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown]: why `search` ?

Comment (RonaldDijkstra) [reply source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown]: The other one didn't work last week.

File: source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [reply source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown]: To see how sharp you are. 

Sharp as a razor.  😄 👍

File: source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown (hunk 2 above)
Comment (RonaldDijkstra) [reply source/blog/nl/2015-05-20-animated-loading-indicator-with-isometric-cubes.html.markdown]: Works now though 👍

**Pull request merged**